components/translation/custom_formats.rst

```diff
@@ -117,3 +117,9 @@ YAML file are dumped into a text file with the custom format::
     $dumper = new MyFormatDumper();
     $dumper->dump($catalogue, array('path' => __DIR__.'/dumps'));
 
+.. versionadded:: 2.8
+    The ability to format a message catalogue without dumping it was introduced in Symfony 2.8.
+
+.. tip::
+
+    To format a message catalogue without dumping it into files use the `formatCatalogue` method.
```

Comment:
you have to use double  backticks to indicate inline code blocks in reStructured Text